src/Mod/Material/StandardMaterial/ABS-Generic.FCMat

```diff
@@ -1,19 +1,26 @@
 ; ABS-Generic
 ; (c) 2013 Juergen Riegel (CC-BY 3.0)
-; information about the content of such cards you can find here:
-; http://www.freecadweb.org/wiki/index.php?title=Material
+; information about the content of such cards can be found on the wiki:
+; https://www.freecadweb.org/wiki/index.php?title=Material
```

Comment:
Why not https://www.freecadweb.org/wiki/Material ?

Reply:
done